django_project/geosight/data/models/context_layer.py

```diff
@@ -99,6 +106,29 @@ class ContextLayer(AbstractEditData, AbstractTerm):
             'that will be used to autogenerate the token.'
         )
     )
+    related_table = models.ForeignKey(
+        RelatedTable,
+        null=True, blank=True,
+        on_delete=models.SET_NULL,
+        help_text=_(
+            'Related table name.'
+        )
+    )
+
+    latitude = models.TextField(
```

Comment:
Could you change to "latitude_field" or "latitiude_key"?
Also please put help text to explain this.
Same with below

And also could you put everything in the field 
config = models.JsonField(
        null=True, blank=True
    )
and put latitude, longitude, datetime and query to config as json?

Just to prevent a long list of new fields

Reply:
No problem in changing naming. But my 2 cents: 
Not sure if using Jsonfields on a relational DB is the best option ... It makes a lot of sense in a related table where we don't know incoming data, but not in a context data model. What do you think?

Reply:
i think we can put it as field
but could you update the field name
and add the help text, so we know the fields basically are for related table type?

Thank you

Reply:
Of course! Done! Thanks!